src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -13,6 +13,23 @@ public class ZeroTouchSearchElement : NodeSearchElement
     {
         private readonly FunctionDescriptor functionDescriptor;
 
+        public override string Description
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(functionDescriptor.Description))
+                    return Dynamo.UI.Configurations.NoDescriptionAvailable;
+
+                return functionDescriptor.Description;
+            }
+            set
+            {
+                if (value == functionDescriptor.Description) return;
+                description = value;
+                OnPropertyChanged("Description");
+            }
+        }
+
```

Comment:
You are right about setter. Setter is excess here. Description for every node is loaded before app runs. It's because we add description in search dictionary. We can reduce time spent on it, if we load description only on demand(mouse over member).
But in this case we don't have an opportunity to find node by its description.